apps/preview/nuxt/pages/showcases/FormFields/FormFieldsRequired.vue

```diff
@@ -0,0 +1,634 @@
+<template>
+  <div class="px-4">
+    <h1 class="mb-4 typography-headline-4 font-bold">Personal information</h1>
+    <form @submit.prevent="sendForm">
+      <label class="flex justify-between typography-text-base cursor-pointer mb-4 gap-2">
+        Don’t display my personal information on a public profile
+        <SfSwitch v-model="personalInformationModelValue" />
+      </label>
+      <label>
+        <span class="typography-label-sm font-medium">Username *</span>
+        <SfInput
+          v-model="usernameModelValue"
+          :required="true"
+          :invalid="usernameIsInvalid"
+          @input="usernameModelValue ? (usernameIsInvalid = false) : (usernameIsInvalid = true)"
+          @blur="usernameModelValue ? (usernameIsInvalid = false) : (usernameIsInvalid = true)"
```

Comment:
Suggestion: isn't that enough?
```suggestion
          @update:model-value="usernameIsInvalid = !usernameModelValue"
```

Reply:
Same for any other component

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2787/commits/361d45dcea22beee7c7f07252163b9882dd2946c